src/tests/locale/test_spelling.py

```diff
@@ -0,0 +1,98 @@
+import polib
+import enchant as ec
+import os
+from enchant.checker import SpellChecker
+from enchant.tokenize import Filter, HTMLChunker
+from shutil import rmtree
+
+class PythonFormatFilter(Filter):
+    def _skip(self, word):
+        if word[:2] == "%(":
+            return True
+        if word[0] == "{":
+            return True
+        return False
+
+# The languages dict maps the language of the po-file to the the dictionary language used by pyenchant
+languages = {
+        "de": "de_DE",
+        "en": "en_US",
+}
+
+# These wordlists in the IGNORES_DIR only need to contain uncapitalized words,
+# the corresponding capitalized words are automatically recognized.
+LOCALES_DIR = os.path.abspath('../../pretix/locale')
+IGNORES_DIR = os.getcwd()
+BUILD_DIR = os.path.abspath('./_build')
+
+try:
+    print('Creating build directory at' + BUILD_DIR + '…')
```

Comment:
for proper insertion of whitespace repace `+` by `,`, otherwise you end up printing something like "…at/foo/bar/builddir".
The `…` can be ommitted, I' say, shortening the print statement.